rheem-tests/src/test/resources/rheem.properties

```diff
@@ -1 +1,2 @@
-spark.driver.host = localhost
\ No newline at end of file
+spark.driver.host = localhost
+rheem.giraph.hdfs.tempdir = hdfs://localhost:9000/result/
```

Comment:
Is this a good setting for the tests? In general, we don't have an HDFS installed during the tests.